flocker/ca/functional/test_validation.py

```diff
@@ -385,3 +389,6 @@ def test_requests_dns(self):
         the REST API when it is accessed via a domain name.
         """
         return self.assert_requests_validates(b"localhost", "control_dns")
+    test_requests_dns.skip = (
```

Comment:
I don't see why this is better than just removing the test, and creating an issue to create it again.